chainer/functions/normalization/decorrelated_batch_normalization.py

```diff
@@ -0,0 +1,305 @@
+from chainer import backend
+from chainer import function_node
+from chainer.utils import argument
+from chainer.utils import type_check
+
+
+def _calc_axis_and_m(x_shape, batch_size, groups):
+    m = batch_size * groups
+    spatial_ndim = len(x_shape) - 2
+    spatial_axis = tuple(range(2, 2 + spatial_ndim))
+    for i in spatial_axis:
+        m *= x_shape[i]
+    return spatial_axis, m
+
+
+class DecorrelatedBatchNormalizationFunction(function_node.FunctionNode):
```

Comment:
A detail and a matter of consistency but do you need the "Function" postfix to the class name? Ditto for the fixed one.